models/homePage.js

```diff
@@ -1,6 +1,17 @@
-var path = require('path');
+const database = require('../utils/database');
 
-exports.cb0 = function (req, res) 
+exports.GetRecentListings = async function (callback) 
```

Comment:
Would be nice to use camel case convention and keep things consistent.

Reply:
See pm